source/docs/software/examples-tutorials/drivesim-tutorial/drivetrain-model.rst

```diff
@@ -132,9 +135,9 @@ You can calculate the measurement noise of your sensors by taking multiple data
         // Create a linear system from our characterization gains.
         frc::LinearSystemId::IdentifyDrivetrainSystem(
           KvLinear, KaLinear, KvAngular, KaAngular),
-        0.7112_m,           // The track width is 0.7112 meters.
```

Comment:
The argument order for C++ is correct: https://first.wpi.edu/wpilib/allwpilib/docs/release/cpp/classfrc_1_1sim_1_1DifferentialDrivetrainSim.html#af783414d0b6f57f46f2fd539e6395df4

Unfortunately, the two languages are not consistent but it would be a breaking change to fix it now.